### DISCUSSION TITLE
Protein q values for spiked proteins

### DISCUSSION THREAD

**Discussion by ahmohamed** [vdemichev/DiaNN#241] [Q&A] [answered]
Hi Vadim,

As part of our QC pipeline, we commonly include spiked-in proteins (mainly ovalbumin, OVA). We noticed that calculation of protein q-values is behaving differently. Below is an experiment with 384 samples, all of which spiked with OVA. 

![image](https://user-images.githubusercontent.com/6384340/141677339-f77148f2-4e98-4d98-b53d-0e31ca0a850b.png)

As you can see, q-value is always 1, despite the detected intensity (as extracted from DIA-NN main report). OVA is already included in the spectral library. One thing to note is that we search each sample individually (not the whole experiment). Any idaes why this is happening?

Cheers.
Ahmed.

**Comment by vdemichev** [comment, marked as answer]
Hi Ahmed,

Try PG.Q.Value? Protein.Q.Value is calculated using proteotypic peptides only, I guess the peptides you use for the spike in are not marked as protoetypic.

Best,
Vadim

**Reply by ahmohamed** [reply]
We did not spike peptides. We spike OVA protein prior to sample digestion. The peptides should be protoetypic, right? They go through the same (tryptic) digestion process as the rest of the proteins in the sample.

**Reply by ahmohamed** [reply]
Actually you're right. Here are the results with PG.Q.Value. However, it seems there's another issue with that as well. Q values in blanks samples (PBS sample, colored red) are consistently non-significant, even though they have higher intensities.

![image](https://user-images.githubusercontent.com/6384340/141702863-587d190c-08ff-4c86-8606-dd0de9aa1232.png)

**Comment by vdemichev** [comment]
Btw, if not using MBR, no need to search runs individually.

**Reply by ahmohamed** [reply]
Thanks. We're searching runs individually as part of a continuous monitoring workflow, i.e, we're searching runs as soon as they are finished.
